corehq/apps/reports/v2/endpoints/case_properties.py

```diff
@@ -28,7 +28,10 @@ class CasePropertiesEndpoint(BaseOptionsEndpoint):
     @property
     @memoized
     def excluded_types(self):
-        return ['name'] + self.report_context.get('existingColumnNames', [])
+        return [
+           'name',
+           '_link',
```

Comment:
Should this still be marked as a private variable?  Or is the underscore prefix used to denote an internal property or something?

Reply:
the prefix is used to denote an internal property. this is coming from the same structure / code as the existing CLE